src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/Tags.scala

```diff
@@ -3,7 +3,14 @@ package uk.gov.nationalarchives.tdr.api.graphql
 import sangria.execution.{BeforeFieldResult, FieldTag}
 import sangria.schema.Context
 import uk.gov.nationalarchives.tdr.api.auth.ValidationAuthoriser.{AuthorisationException, continue}
+import uk.gov.nationalarchives.tdr.api.graphql.fields.ConsignmentFields.Consignment
+import uk.gov.nationalarchives.tdr.api.graphql.fields.TransferAgreementFields.AddTransferAgreementInput
 
+import scala.util.{Failure, Success}
+import scala.concurrent._
+import ExecutionContext.Implicits.global
+import scala.concurrent.duration._
```

Comment:
There's a couple of unused imports here.

Reply:
Fixed